tools/buildPackage.js

```diff
@@ -112,10 +116,14 @@ cmds = cmds.concat([
 
 console.log('Building version ' + VersionInfo.braveVersion + ' in ' + buildDir + ' with Electron ' + VersionInfo.electronVersion)
 
+// Only way to give a different CFBundleName is to modify the 'app' name passed to electron-builder,
+// which also changes the output path.
+const tempModifiedAppName = (isDarwin && process.env.BUNDLE_NAME) ? process.env.BUNDLE_NAME : null
```

Comment:
@bsclifton did we modify the build environment to contain this `BUNDLE_NAME` variable?

Reply:
OMG - no, I didn't do that (yet). Thanks for the reminder 😄 (I'll mark this as resolved once it happens)

Reply:
Looks like it was already there the whole time 😄 👍 (I think you added it- thanks!)
<img width="316" alt="screen shot 2018-12-07 at 9 29 42 am" src="https://user-images.githubusercontent.com/4733304/49659997-bc402880-fa02-11e8-9df9-9a59581c5629.png">